src/app/Components/LanguageInfo.svelte

```diff
@@ -0,0 +1,58 @@
+<script lang="ts">
+  import InputField from "./InputField.svelte";
+  export let properties: any;
+</script>
+
+<style>
+  .language__info {
```

Comment:
**[Question]**: I love the BEM naming convention, but if we are using CSS these doesn't make a different

Reply:
Yeah, BEM doesn't really make sense if we're using scoped CSS, which is the default in Svelte. I don't see the need for any changes right now, but something to keep in mind.